tests/tasks/create_test_interfaces_with_dhcp.yml

```diff
@@ -43,8 +43,17 @@
       nmcli d set testbr managed false
     fi
     ip link set testbr up
-    ip addr add 192.0.2.1/24 dev testbr
-    ip -6 addr add 2001:DB8::1/32 dev testbr
+    timer=0
+    # The while loop following is a workaround for the bug
+    # https://bugzilla.redhat.com/show_bug.cgi?id=2079054
```

Comment:
is this workaround going to be removed once NM is fixed? In this case, it seems useful to actually track that this workaround exists so we will also remove it eventually.

Reply:
Updated.